text_recognizer/data/iam_synthetic_paragraphs.py

```diff
@@ -47,9 +47,9 @@ def prepare_data(self, *args, **kwargs) -> None:
         for split in ["train", "val", "test"]:
             rank_zero_info(f"Cropping IAM line regions and loading labels for {split} data split...")
             crops, labels = line_crops_and_labels(iam, split)
-    
-            crops = [resize_image(crop, IMAGE_SCALE_FACTOR) for crop in crops]  # TODO: iam_syn does this but iam_para does not. So when does iam_para resize?
```

Comment:
yep, that's one of the root causes of the resizing bug i introduced when refactoring. we're inconsistent with when we do resizing.